internal/pkg/jwt/types.go

```diff
@@ -0,0 +1,52 @@
+package jwt
+
+import (
+	"context"
+	"crypto/rsa"
+	"github.com/xmidt-org/ears/pkg/tenant"
+	"net/http"
+	"sync"
+)
+
+const (
+	MissingToken           = "missing token"
```

Comment:
I thought our ears convention is to make each msg a specific error instead of one error with different messages

Reply:
yes, this is legacy stuff from the common library, will fix

Reply:
actually i think this is ok, because we do have a JWTAuthError type defined in errors.go and just use the strings to distinguish subtle flavors - for clarity i moved the constants to error.go